extensions/wikia/PortableInfobox/js/PortableInfobox.js

```diff
@@ -33,9 +33,13 @@
 
 			$collapsibleGroups.each( function( index, group ) {
 				var $group = $collapsibleGroups.eq(index),
-					$header = $group.find('.pi-header');
+					$header = $group.find('.pi-header'),
+					title = $group.hasClass('pi-collapse-closed') ? 'Open' : 'Close';
```

Comment:
Those are hardcoded english words and we have rule of not hardcoding such stuff - use i18n mechanism if you can.